app/vmui/packages/vmui/src/components/ExploreAlerts/Notifier/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "preact/compat";
+import "./style.scss";
+import { Target } from "../../../types";
+
+interface NotifierProps {
+  targets: Target[]
+}
+
+const Notifier: FC<NotifierProps> = ({
+  targets,
+}) => {
+
+  return (
+    <div className="vm-explore-alerts-notifier">
+      <table>
+        <thead>
+          <tr>
+            <th className="col-small">Address</th>
+            <th>Labels</th>
+          </tr>
+        </thead>
+        <tbody>
+          {targets.map(target => (
+            <tr key={target.address}>
+              <td className="col-small">{target.address}</td>
+              <td className="badge-container">
+                {Object.entries(target.labels || {}).map(([name, value]) => (
+                  <span
+                    className="badge"
+                    key={name}
+                  >{`${name}: ${value}`}</span>
+                ))}
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
```

Comment:
Can we use instead of this `Table` component from [`src/components/Table/Table.tsx`](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/777f05efcc3ef3440c8821861dbc91357dd9c586/app/vmui/packages/vmui/src/components/Table/Table.tsx)?

Reply:
haven't checked it, will take a look

Reply:
just checked Table component implementation and it's use cases, I'm not sure it's good for this data, also not sure if table is the best choice for this content at all, maybe for rule annotations only. wanted to reuse something similar to group logs header

<img width="1000" alt="image" src="https://github.com/user-attachments/assets/62e20f92-fa03-432c-9b09-9047ac1400fa" />